daily-challenges/2024-09/26/hunghoang3110.spec.ts

```
@@ -0,0 +1,34 @@
+test('filter student database', async ({ page }) => {
+
+  // Navigate to data page
+  await page.goto('https://material.playwrightvn.com/021-import-export.html');
+  await expect(page).toHaveTitle('Quản Lý Dữ Liệu Học Sinh');
+  const rows = await page.locator('tbody > tr').count()
+
+  // Filter and verify
+  await page.getByRole('combobox').selectOption('lop')
+  await page.getByPlaceholder('Nhập nội dung tìm kiếm...').fill('A4');
+  await page.getByRole('button', { name: 'Tìm kiếm' }).click();
+  let visibleRows = await page.locator('tbody > tr:not([style*="display: none"])');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'visibleRows' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)